backend/flindt/round/manager.py

```diff
@@ -153,6 +155,8 @@ def do_all_users_have_a_role(self):
         return users_without_role
 
     def do_all_users_have_a_slack_id(self):
+        if settings.SILENT_RUN:
```

Comment:
This change is not needed here. If you run into an error, please put it on the more appropiate place. This function simply returns all users without a slack id, but it does not connect to Slack in any way. So DRY_RUN should only be used on places where actual Slack communication is used.